include/unifex/just.hpp

```diff
@@ -110,6 +122,18 @@ namespace _just_cpo {
 } // namespace _just_cpo
 using _just_cpo::just;
 
+namespace _just_done_cpo {
+  inline const struct just_done_fn {
+    template <typename... Values>
+    constexpr auto operator()(Values&&... values) const
```

Comment:
Why does `just_done()` accept values when it will never complete with a value?

Reply:
Clunky agreed. It was the simplest hack to infer a matching type. I could have templated the CPO.

Reply:
Ok, it is no more. There is a TODO instead.